style.css

```diff
@@ -0,0 +1,182 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    min-width: 100vw;
+    min-height: 100vh;
+    position: relative;
+    left: 0;
+    top: 0;
+    background-color: lightblue;
+}
+
+/* header  */
+.heading {
```

Comment:
you only have one h1 as far as I can see. You could directly select `h1` here rather than having to select it by a class name